ush/python/pygfs/jedi/jedi.py

```diff
@@ -216,18 +218,45 @@ def get_bias_dict(self, task_config: AttrDict) -> Dict[str, Any]:
                 obfile = ob['obs bias']['input file']
                 obdir = os.path.dirname(obfile)
                 basename = os.path.basename(obfile)
-                prefix = '.'.join(basename.split('.')[:-2])
-                for file in ['satbias.nc', 'satbias_cov.nc', 'tlapse.txt']:
-                    bfile = f"{prefix}.{file}"
-                    copylist.append([os.path.join(task_config.COM_ATMOS_ANALYSIS_PREV, bfile), os.path.join(obdir, bfile)])
-                    # TODO: Why is this specific to ATMOS?
+                prefix = '.'.join(basename.split('.')[:-3])
+                bfile = f"{prefix}.{bias_file}"
+                tar_file = os.path.join(obdir, bfile)
+                copylist.append([os.path.join(task_config.VarBcDir, bfile), tar_file])
+                break
```

Comment:
is this break correct?

Reply:
The bias correction tarball has a `prefix` and path, `obdir`.  The ob loop extracts this information when it finds an `obs bias` in `observers`.  Once we have the `prefix` and `obdir` path we can construct the `copylist` entry to copy the bias correction tarball from the source to the destination.  Since the tarball contains all the bias correction files we only need to copy a single file.  Hence breaking out of the loop once the {source, destination} pair is appended to `copylist.  

If there's a better way to do this, I'm open to suggestions.

Reply:
Sorry, I was confused at what this is doing.

So this is saying, if there is at least one file with `obs bias`, create the tar ball. 

I'm not 100% convinced this is future proof, as there would be no way to distinguish between rad_bias and acft_bias in this case, so it may try to copy rad when only acft or vice versa. However, maybe we just cross that bridge when we come to it?

Reply:
We pass the root name of the bias correction file into the `get_bias_dict` method via `bias_file`.  For radiance bias correction `bias_file` is set in the parent script to be `rad_varbc_params.tar`.  The parent script executes `FileHandler` on the returned `bias_dict`.  This is followed by an `extract_tar` to extract the _satellite_sensor_ specific radiance bias correction files in the tarball.

From what I have heard the aircraft bias correction will be a single file for all tail numbers.  We would invoke `get_bias_dict` with a different `bias_file`.  The parent script would pass the returned `bias_dict` to `FileHandler`.  There's no need to invoke `extract_tar` if the aircraft bias correction file is just a file and not a tarball.

I acknowledge that what's in this PR may not work out of the box for aircraft bias correction.  My preference is to address this via a separate g-w issue and PR.

Reply:
Thanks, @CoryMartin-NOAA.  I can test aircraft bias correction from the scripting side when we're ready.